plugins/woocommerce/src/Internal/DataStores/Orders/CustomOrdersTableController.php

```diff
@@ -197,21 +204,25 @@ final public function init( OrdersTableDataStore $data_store, DataSynchronizer $
 	 * @return bool True if the feature is visible.
 	 */
 	public function is_feature_visible(): bool {
-		return $this->is_feature_visible;
+		return $this->features_controller->feature_is_enabled( 'custom_order_tables' );
 	}
 
 	/**
 	 * Makes the feature visible, so that dedicated entries will be added to the debug tools page.
+	 *
+	 * @throws \Exception This method shouldn't be used anymore, see the FeaturesController class.
 	 */
 	public function show_feature() {
-		$this->is_feature_visible = true;
+		throw new \Exception( 'The visibility of the custom orders table feature is now handled by the WooCommerce features engine. See the FeaturesController class, or go to WooCommerce - Settings - Advanced - Features.' );
```

Comment:
These probably shouldn't be exceptions, but just no-ops with a `doing_it_wrong` message. Many folks who are currently testing will have them enabled.